text/0156-columnar-encoding.md

```diff
@@ -0,0 +1,2195 @@
+of metadata duplication observed in the existing protocol.
+
+The attributes `aggregation_temporality` and `is_monotonic` are attached as metadata to the corresponding column
+metric (for sum, histogram, and exponential histogram).
+
+Multivariate metrics can easily be represented as multiple metrics columns sharing the same `start_time_unix_nano`,
+and `time_unix_nano`, attributes, ... The name of each multi-variate metric must be unique in the scope of the
+corresponding metric type (gauge, sum, histogram, exp_histogram, summary).
+
+> Open question: `aggregation_temporality` and `is_monotonic` are in this proposal represented as column metadata, but
+> they could also be represented as 2 distinct columns if needed. Feedback from the community on this choice will be
+> appreciated.
+
+#### Logs Payload
+
+Although simpler, a logs 'OtlpArrowPayload' takes a similar approach.
+
+| Column name                  | Column type | Required | Description                                                                                                     |
+|------------------------------|-------------|----------|-----------------------------------------------------------------------------------------------------------------|
+| `resource`                   | `struct`    | No       | Structure regrouping the fields of the resource                                                                 |
+| __`attributes`               | `struct`    | No       | Structure regrouping the fields of the resource attributes                                                      |
+| ____`[key]`                  | `dynamic`   | No       | A set of attributes, see [attribute section](#labelattribute-representation) for more details                   |
+| __`dropped_attributes_count` | `uint32`    | No       | The number of resource dropped attributes.                                                                      |
+| __`schema_url`               | `string`    | No       | Schema url.                                                                                                     |
+| `instrumentation_library`    | `struct`    | No       | Structure regrouping the fields of the instrumentation library                                                  |
+| __`name`                     | `string`    | No       | Name of the instrumentation library                                                                             |
+| __`version`                  | `string`    | No       | Version of the instrumentation library                                                                          |
+| `time_unix_nano`             | `uint64`    | Yes      | The time when the event occurred.                                                                               |
+| `severity_number`            | `uint8`     | Yes      | The severity number.                                                                                            |
```

Comment:
The severity number for the undefined case is set to zero. It is commented on the proto file that it should not be used https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto#L79 - should this field be marked as not required?

Reply:
Agree

Reply:
Updated in the OTEP and in the reference implementation.